core/tests/puppeteer-acceptance-tests/user-utilities/curriculum-admin-utils.ts

```diff
@@ -0,0 +1,703 @@
+    await this.page.bringToFront();
+
+    for (let i = 0; i < questionCount; i++) {
+      await this.createQuestion();
+    }
+    await this.assignSkillToSubtopic();
+  }
+
+  /**
+   * Function for creating a question in the skill editor page.
+   */
+  async createQuestion(): Promise<void> {
+    await this.openSkillEditor();
+    await this.clickOn(createQuestionButton);
+    await this.clickOn(textStateEditSelector);
+    await this.page.waitForSelector(richTextAreaField, {visible: true});
+    await this.type(richTextAreaField, 'Add 1+2');
+    await this.page.waitForSelector(`${saveContentButton}:not([disabled])`);
+    await this.clickOn(saveContentButton);
+
+    await this.clickOn(addInteractionButton);
+    await this.page.waitForSelector(modalDiv, {visible: true});
+    await this.clickOn(interactionNumberInputButton);
+    await this.clickOn(saveInteractionButton);
+    await this.page.waitForSelector('oppia-add-answer-group-modal-component', {
+      visible: true,
+    });
+    await this.clickOn(responseRuleDropdown);
+    await this.clickOn(equalsRuleButtonText);
+    await this.type(floatTextField, '3');
+    await this.clickOn(answersInGroupAreCorrectToggle);
+    await this.clickOn(saveResponseButton);
+    await this.page.waitForSelector(modalDiv, {hidden: true});
+
+    await this.clickOn(defaultFeedbackTab);
+    await this.clickOn(openOutcomeFeedBackEditor);
+    await this.clickOn(richTextAreaField);
+    await this.type(richTextAreaField, 'The answer is 3');
+    await this.clickOn(saveOutcomeFeedbackButton);
+
+    await this.clickOn(addHintButton);
+    await this.page.waitForSelector(modalDiv, {visible: true});
+    await this.type(richTextAreaField, '3');
+    await this.clickOn(saveHintButton);
+    await this.page.waitForSelector(modalDiv, {hidden: true});
+
+    await this.clickOn(addSolutionButton);
+    await this.page.waitForSelector(modalDiv, {visible: true});
+    await this.page.waitForSelector(answerTypeDropdown);
+    await this.page.select(answerTypeDropdown, 'The only');
+    await this.page.waitForSelector(solutionFloatTextField);
+    await this.type(solutionFloatTextField, '3');
+    await this.page.waitForSelector(`${submitAnswerButton}:not([disabled])`);
+    await this.clickOn(submitAnswerButton);
+    await this.type(richTextAreaField, '1+2 is 3');
+    await this.page.waitForSelector(`${submitSolutionButton}:not([disabled])`);
+    await this.clickOn(submitSolutionButton);
+    await this.page.waitForSelector(modalDiv, {hidden: true});
+
+    await this.clickOn(saveQuestionButton);
+  }
+
+  /**
+   * Function for navigating to the contributor dashboard page.
+   */
+  async navigateToCreatorDashboardPage(): Promise<void> {
+    await this.page.bringToFront();
+    await this.goto(creatorDashboardUrl);
+  }
+
+  /**
+   * Function for creating an exploration as a curriculum admin.
+   */
+  async createExploration(state: string, title: string): Promise<string> {
```

Comment:
"state" is unclear. Do you mean "initialContentText" or something like that? (Also, if that's the case, the argument should probably come after the title.)